### DISCUSSION TITLE
Remove Vue 2 support in Instantsearch v5?

### DISCUSSION THREAD

**Discussion by Hawxy** [algolia/instantsearch#6413] [Ideas]
With a new major version being worked on, and Vue 2 being EoL almost 12 months ago, is it possible support can finally be removed? It's the main barrier at the moment for a proper Vue 3 rewrite with Typescript support per previous discussions.

**Comment by Haroenv** [comment]
This is not an initial goal of the major version, mainly because of the amount of work it implies. Are you interested in contributing to this? If so we can check the numbers to ensure dropping vue 2 wouldn't cause too many stranded users.
